server/server.go

```diff
@@ -486,6 +486,7 @@ func (s *Server) IsClosed() bool {
 // Run runs the pd server.
 func (s *Server) Run() error {
 	go systimemon.StartMonitor(s.ctx, time.Now, func() {
+		defer logutil.LogPanic()
```

Comment:
We have already had a defer in StartMonitor

Reply:
done